<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddStudent</class>
 <widget class="QWidget" name="FormAddStudent">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Student</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayoutAddStudent">
     <item row="0" column="0">
      <widget class="QLabel" name="lblAddStudentName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineAddStudentName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblAddStudentGrade">
       <property name="text">
        <string>Grade</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineAddStudentGrade"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineAddStudentAge"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAddStudent">
     <item>
      <widget class="QPushButton" name="btnSaveStudent">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearStudent">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelStudent">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
